github/code-scanning.go

```diff
@@ -18,11 +18,17 @@ import (
 // GitHub API docs: https://docs.github.com/en/free-pro-team@latest/rest/reference/code-scanning/
 type CodeScanningService service
 
+type Tool struct {
```

Comment:
While you are touching this file, could you please add a struct comment for both `Tool` and `Alert` describing what they represent?